packages/nextjs/components/Header.tsx

```diff
@@ -101,7 +119,34 @@ export const Header = () => {
           <HeaderMenuLinks />
         </ul>
       </div>
+
       <div className="navbar-end flex-grow mr-4">
+        <div className="flex">
+          {address ? (
+            isInAllowList ? (
+              isCheckedIn ? (
+                <div
+                  className="tooltip tooltip-bottom mr-5"
+                  data-tip="You are in the allowlist and you have checked in"
+                >
+                  <button className="btn-active bg-transparent text-2xl">✔️</button>
```

Comment:
It is kind of not properly visible in dark mode : 
![Screenshot 2024-06-06 at 9 23 55 AM](https://github.com/BuidlGuidl/batch6.buidlguidl.com/assets/80153681/ce69720d-9044-43f7-99b4-93c6ab1ad64b)


Also maybe its nice idea to use Heroicons components instead of emojis  ? 
![Screenshot 2024-06-06 at 9 22 06 AM](https://github.com/BuidlGuidl/batch6.buidlguidl.com/assets/80153681/fd2be79e-3499-4b98-b743-0f7c674dbbad)

Since emoji's renders differently on different systems eg mac and windows etc

same for other emojis too